src/gt4py/backend/gtc_backend/gtcpp/backend.py

```diff
@@ -119,13 +120,26 @@ def visit_FieldDecl(self, node: gtcpp.FieldDecl, **kwargs):
                     sid_ndim=sid_ndim,
                 )
             else:
+                backend_cls = globals()[
```

Comment:
Let's pass this information to this generator and not try to reverse engineer it.

Reply:
Oh man, I hoped you would overlook this ;) done.